.circleci/config.yml

```diff
@@ -93,6 +93,13 @@ jobs:
       - run:
           name: Run tests
           command: npx bazel test //lambdas:unit_test --test_output=all
+  build-commons:
```

Comment:
Maybe others should depend on build-commons?

Reply:
Let's add this job as part of the requirements of `accept-publish-docker` in line 206 to make sure tests are running properly for this package.

Reply:
I just realized we need to add the same thing for commons/utils. I'll do it in #95

Reply:
Or... maybe we could include it as a step of this build. What do you think?

Reply:
Done :)